packages/bezier-react/src/components/Forms/SegmentedControl/SegmentedControl.tsx

```diff
@@ -40,26 +42,23 @@ function SegmentedControlItemListImpl<
   style: styleProp,
   className: classNameProp,
   ...rest
-}: SegmentedControlItemListProps<Type, Value>, forwardedRef: React.Ref<HTMLDivElement>) {
-  const [selectedElement, setSelectedElement] = useState<HTMLButtonElement | null>(null)
+}: SegmentedControlItemListProps<Type, Value>) {
+  const [index, setIndex] = useState<number | null>(null)
```

Comment:
```suggestion
  const [selectedItemIndex, setSelectedItemIndex] = useState<number | null>(null)
```

가독성 향상을 위해 변수 의미가 더 명확해지면 좋을 거 같아요.